src/CrossFilter.js

```diff
@@ -81,6 +85,7 @@ export default class CrossFilter {
   initCrossFilterForAsync(dataConnector, dataTables) {
     this._dataConnector      = dataConnector
     this._cache              = new ResultCache(dataConnector) // this should gc old cache?
+    this._groupAllcache      = new ResultCache(dataConnector)
     this._dataTables         = Array.isArray(dataTables) ? dataTables : [dataTables]
```

Comment:
If there is a GroupAll cache, it should be encapsulated by GroupAll and managed by it. Also, _groupAllCache vs _groupAllCache